web/email.ex

```diff
@@ -17,4 +17,13 @@ defmodule Community.Email do
     |> subject("Thank you for posting")
     |> render("job_posted.text", job: job)
   end
+
+  def contact_form(data) do
+    new_email
+    |> to(data[:to])
```

Comment:
I agree completely. For some reason I thought that you couldn't use the dot notation unless it was a struct but it seems that only true if the strings are keys. Thanks!